### PR TITLE
Adding lab-01

### PR DESCRIPTION
**Pull request by VedikaSrivastava**

### CODE REVIEW

File: labs/lab-01_README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+## Helpful Resources
+
+- [Getting Started with Python](https://www.python.org/about/gettingstarted/)
+- [Getting Started with Git](https://docs.github.com/en/get-started/getting-started-with-git)
+
+![image](https://github.com/VedikaSrivastava/CS506-labs-Fall2023/assets/83489280/217b0f8f-1366-4e74-8d8a-f99dcc01e9ec)
+
+
+### Git/GitHub Workflow
```
Comment (gallettilance) [inline labs/lab-01_README.md:15]:
```suggestion
### Lab Instructions

Please complete the items below in order.
```

File: labs/lab-01_README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# Becoming familiar with Git/Github
```
Comment (gallettilance) [inline labs/lab-01_README.md:1]:
```suggestion
# Git / GitHub Lab
```

File: labs/lab-01_README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+     git remote add upstream <url of repo>
+     ```
+
+4. [**Create and Checkout a New Branch**](https://github.com/Kunena/Kunena-Forum/wiki/Create-a-new-branch-with-git-and-manage-branches)
+   - Create a new branch for your code changes. Replace `<branch-name>` with an appropriate branch name describing the component you are working on:
+     ```shell
+     git checkout -b <branch-name>
+     ```
+
+5. **Implement Your Component**
```
Comment (gallettilance) [inline labs/lab-01_README.md:48]: maybe change this to something more like "make a change"

File: labs/lab-01_README.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+8. [**Create a Pull Request**](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+   - Go to your forked repository on GitHub.
+   - You should see a banner with a "Compare & pull request" button for the branch you just pushed. Click on it.
+   - Provide a descriptive title and any additional information about your changes in the comment box.
+   - Click the "Create pull request" button.
+
+Congratulations! You've successfully forked a repository, made code contributions, and created a pull request using Git.
+
+### [Multi Author Commit](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors)
```
Comment (gallettilance) [inline labs/lab-01_README.md:76]: Nit: add a newline

**Review by gallettilance — CHANGES_REQUESTED** [review]
Added some suggestions. Otherwise looks great!

**Comment by VedikaSrivastava** [comment]
I have made the suggested changes

**Pull request merged**